app/src/main/java/org/mozilla/fenix/tabtray/TabTrayFragment.kt

```diff
@@ -201,6 +217,29 @@ class TabTrayFragment : Fragment(R.layout.fragment_tab_tray), TabsTray.Observer,
         navController.nav(R.id.tabTrayFragment, directions)
     }
 
+    private fun registerCollectionStorageObserver() {
+        requireComponents.core.tabCollectionStorage.register(collectionStorageObserver, this)
+    }
+
+    private fun showCollectionSnackbar() {
+        if (view != null) {
```

Comment:
Is there a way we can better solidify view so we aren't checking for it everywhere?

Reply:
I've been doing this
```
view?.also {
    
}
```